src/Compilers/CSharp/Portable/Symbols/DiscardSymbol.cs

```diff
@@ -15,6 +15,7 @@ public DiscardSymbol(TypeSymbol type)
 
         ITypeSymbol IDiscardSymbol.Type => Type;
         public TypeSymbol Type { get; }
+        public Nullability Nullability => Nullability.NotComputed;
```

Comment:
>public Nullability Nullability [](start = 8, length = 30)

Consider using explicit implementation if this property is only intended to be used through the interface.

Same question for `Symbol` implementations of each `Nullability` property.